lib/cdo/shared_constants/curriculum/shared_course_constants.rb

```diff
@@ -68,6 +68,9 @@ module SharedCourseConstants
     csf_international: 'CS Fundamentals International',
     math: 'Math',
     twenty_hour: '20-hour',
-    other: 'Other'
+    other: 'Other',
+    pl_self_paced: 'Self Paced Professional Learning',
```

Comment:
Nit: Is this being marketed as "Self-Paced PL" or "Self Paced PL"? I think grammatically the hyphen is there but not sure we're doing that

Reply:
Just saw this comment. I will update in a future PR